ui/components.d.ts

```diff
@@ -55,8 +51,6 @@ declare module 'vue' {
     NTree: typeof import('naive-ui')['NTree']
     NUpload: typeof import('naive-ui')['NUpload']
     NUploadDragger: typeof import('naive-ui')['NUploadDragger']
-    NUploadFileList: typeof import('naive-ui')['NUploadFileList']
-    NUploadTrigger: typeof import('naive-ui')['NUploadTrigger']
     NWatermark: typeof import('naive-ui')['NWatermark']
     RouterLink: typeof import('vue-router')['RouterLink']
     RouterView: typeof import('vue-router')['RouterView']
```

Comment:
I can examine the code for specific issues but since you did not specify which parts of the code you want checked, I will need to review these parts manually or ask for more details. Please provide the exact codes where there are discrepancies or improvements needed, and I would be happy to help optimize them in English.